lib/run/export-file.js

```diff
@@ -77,9 +82,25 @@ module.exports = function (options, done) {
                     `error creating path for file "${path.unparsed}" for ${options.name || 'unknown-source'}`));
             }
 
-            fs.writeFile(path.unparsed, content, function (err) {
-                done(_.set(err, 'help',
-                    `error writing file "${path.unparsed}" for ${options.name || 'unknown-source'}`), path);
+            fs.stat(path.unparsed, function (error, stat) {
+                var target;
+
```

Comment:
Error handling? Not sure what to do here. Need to decide. Suggestions?

Reply:
If there's an error (the path doesn't exist), then it gets written to as usual. This could be a place to add debug logs, though.